src/completions.jl

```diff
@@ -42,6 +48,20 @@ function basecompletionadapter(line, mod, force)
       continue
     end
   end
+  
+  # completions from the local code block:
+  for c in locals(text, lineNumber, column)
+    if startswith(c, pre)
```

Comment:
Seems better to special case `pre == ""`, since otherwise `PathCompletions` get suppressed:
![image](https://user-images.githubusercontent.com/40514306/64124965-69559f00-cde3-11e9-9802-6d12485e97db.png)

Reply:
Good call.